pehlaschoollibrary/Classes/Common/Controls/CompletePopup.cpp

```diff
@@ -174,29 +174,29 @@ void CompletePopup::show(float delay, std::function<void(void)> callback) {
     
     scheduleOnce([](float) {
         SoundEffect::wowEffect().play();
-     
     }, delay+0.1, "wowSound");;
-    
-    
-    
-    
- 
+
+
+
+
+
     _callback = callback;
     
 }
 
 void CompletePopup::dismiss() {
     Size winSize = getContentSize();
+    //removed the delayed transition after tapping the Complete Popup
     auto seq = Sequence::create(
-                                EaseIn::create(MoveBy::create(0.1, Vec2(0, -30)), 2.0),
-                                EaseOut::create(MoveTo::create(0.2, Vec2(0, winSize.height)), 2.0),
+                                EaseIn::create(MoveBy::create(0.0, Vec2(0, -30)), 2.0),
```

Comment:
Check if the update causes an issue when the page fades out

Reply:
Checked not causing any issue.

Reply:
Please do not change anything without any reason. You cannot predict what might fail in future and then it will become difficult to backtrack.

Reply:
EaseIn and EaseOut with duration(0.1 and 0.2) was making the "Complete" pop up to fade out when the same book was reopened after the completion and the fading out transition can be observed. To remove the delayed transition of Complete pop up on reopening the book it was set to 0.0